tests/lax_vmap_test.py

```diff
@@ -594,6 +594,41 @@ def fun(operand, tangents):
       for bdims in all_bdims(shape, shape):
         self._CheckBatching(fun, 3, bdims, (shape, shape), (dtype, dtype), rng)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_dtype={}_padding={}".format(np.dtype(dtype).name,
+                                                      padding),
+       "dtype": dtype, "padding": padding, "rng_factory": rng_factory}
+      for dtype in float_dtypes
+      for padding in ["VALID", "SAME"]
+      for rng_factory in [jtu.rand_small]))
+  def testSelectAndScatterAdd(self, dtype, padding, rng_factory):
+    rng = rng_factory(self.rng())
+    all_configs = itertools.chain(
+        itertools.product(
+            [(4, 6)],
+            [(2, 1), (1, 2)],
+            [(1, 1), (2, 1), (1, 2)]),
+        itertools.product(
+            [(3, 2, 4, 6)], [(1, 1, 2, 1), (2, 1, 2, 1)],
+            [(1, 2, 2, 1), (1, 1, 1, 1)]))
```

Comment:
Would it make sense to move this into the parameters of the test case?

Reply:
Done.